sdcm/provision/aws/configuration_script.py

```diff
@@ -10,10 +10,13 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2021 ScyllaDB
+import logging
 
 from sdcm.provision.aws.utils import network_config_ipv6_workaround_script, configure_eth1_script
 from sdcm.provision.common.configuration_script import ConfigurationScriptBuilder
 
+LOGGER = logging.getLogger()
```

Comment:
Why logger is added?
This change is already huge. All the not needed stuff must be avoided.

Reply:
redundant. removed